src/Responses/AbstractResponse.php

```diff
@@ -53,8 +53,8 @@
      */
     public function __construct($response)
     {
+        $this->responseAsDom = new DOMDocument();
```

Comment:
DOMDocument is created in any case, so other methods depending on and object work.